src/data/polynomial/eval.lean

```diff
@@ -447,6 +447,47 @@ lemma map_prod {ι : Type*} (g : ι → polynomial R) (s : finset ι) :
   (∏ i in s, g i).map f = ∏ i in s, (g i).map f :=
 eq.symm $ prod_hom _ _
 
+lemma map_sum {ι : Type*} (g : ι → polynomial R) (s : finset ι) :
+  (∑ i in s, g i).map f = ∑ i in s, (g i).map f :=
+eq.symm $ sum_hom _ _
+
+lemma map_support_is_subset (p : polynomial R) : (map f p).support ⊆ p.support :=
+begin
+  intros x,
+  simp only [mem_support_iff, classical.not_not] at *,
```

Comment:
```suggestion
  simp only [mem_support_iff],
```
`contrapose!` will take care of the rest here.

Reply:
Fixed.